src/effects/VST3/internal/x11/RunLoop.cpp

```diff
@@ -0,0 +1,119 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file RunLoop.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include "RunLoop.h"
+
+using namespace internal::x11;
+
+IMPLEMENT_FUNKNOWN_METHODS(RunLoop, Steinberg::Linux::IRunLoop, Steinberg::Linux::IRunLoop::iid);
+
+gboolean RunLoop::OnEvent(GIOChannel *source, GIOCondition condition, gpointer data)
+{
+   auto handler = reinterpret_cast<Steinberg::Linux::IEventHandler*>(data);
```

Comment:
`gpointer` is an alias for `void *` so `static_cast` is appropriate, and less surprising to read